### PR TITLE
NF: Lint fixes

### PR DESCRIPTION
**Pull request by david-allison-1**
Various lint fixes, all automated by Android Studio, but visually confirmed.

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.java [hunk 1]
```diff
@@ -126,20 +126,13 @@ public void onAttach(Context context) {
...
-                case R.id.studyoptions_start:
-                    Timber.i("StudyOptionsFragment:: start study button pressed");
-                    if (mCurrentContentView != CONTENT_CONGRATS) {
-                        openReviewer();
-                    } else {
-                        showCustomStudyContextMenu();
-                    }
-                    return;
-                default:
+    private View.OnClickListener mButtonClickListener = v -> {
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.java:129]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5364558)

**Comment by timrae** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity decreasing per file
==============================
+ AnkiDroid/src/main/java/com/ichi2/utils/DeckNameComparator.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5364558&bid=17579291)

**Comment by timrae** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ AnkiDroid/src/main/java/com/ichi2/utils/DeckNameComparator.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5364558&bid=17579291)

**Comment by mikehardy** [comment]
released as 2.10alpha70 as soon as google finishes processing it

**Pull request merged**